src/App.js

```diff
@@ -21,6 +25,10 @@ function App() {
 
   return (
     <Switch>
+      <Route path="/setting" exact component={AccountSetting} />
```

Comment:
I think routes should go in routes list. This lets you manipulate them programmatically. You can generate them, import/export them, filter and join them etc.
You won't need these features if you have 10 routes, but you want them when you scale up.